charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml

```diff
@@ -46,6 +46,10 @@ spec:
   sha: {{ .Values.prometheus.prometheusSpec.image.sha }}
   {{- end }}
 {{- end }}
+{{- if .Values.prometheus.prometheusSpec.additionalArgs }}
+additonalArgs:
```

Comment:
The typing mistake on line 50 should be corrected.

Reply:
fixed the spelling mistake